pallets/phala/src/compute/wrapped_balances.rs

```diff
@@ -384,12 +384,12 @@ pub mod pallet {
 			cid: CollectionId,
 		) -> DispatchResult {
 			let mut account_status =
-				StakerAccounts::<T>::get(who).ok_or(Error::<T>::StakerAccountNotFound)?;
+				StakerAccounts::<T>::get(who).unwrap_or_default();
```

Comment:
Could you explain why it can be empty?

Reply:
When a vault owner gain his owner shares and withdraw, its account will gain W-Pha but not get it by `wrap`. So the acount_status will be empty.

Reply:
Got it. No problem